src/Jackett.Common/Definitions/unleashthecartoons.yml

```diff
@@ -0,0 +1,141 @@
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: id
+    options:
+      id: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: account-login.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    returnto: /
+  error:
+    - selector: div.myFrame-caption:contains("Error")
+      message:
+        selector: div.myFrame-content
+    - selector: div.myFrame-caption:contains("Access denied")
+      message:
+        selector: div.myFrame-content
+  test:
+    path: /
+    selector: a[href="account-logout.php"]
+
+search:
+  paths:
+    # https://unleashthecartoons.world/torrents-search.php?search=&cat=0&incldead=1&freeleech=0&inclexternal=0&lang=0&sort=id&order=desc
+    - path: torrents-search.php
+  inputs:
+    search: "{{ .Keywords }}"
+    # does not support multi cats searching so defaulting to all.
+    cat: 0
+    # 0 active, 1 incldead, 2 onlydead
+    incldead: 1
+    freeleech: "{{ if .Config.freeleech }}2{{ else }}0{{ end }}"
+    # 0 Local/External, 1 Local Only, 2 External Only
+    inclexternal: 0
+    # 0 all, 1 Romanian, 2 English, 3 Romanian/English
+    lang: 0
+    sort: "{{ .Config.sort }}"
+    order: "{{ .Config.type }}"
+    # does not support multicategory filtering
+    # does not support imdbid searches, does not return imdb link in results
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\.", " "]
+    - name: re_replace
+      args: ["\\s+", " "]
+    - name: re_replace
+      args: ["(\\w+)", "+$1"] # prepend + to each word
+
+  rows:
+    selector: table.ttable_headinner tr.t-row
+
+  fields:
+    category:
+      text: 1
+    category|noappend:
+      selector: a[href*="cat="]
+      attribute: href
+      optional: true
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: a[href^="torrents-details.php?id="]
+    details:
+      selector: a[href^="torrents-details.php?id="]
+      attribute: href
+      filters:
+        - name: replace
+          args: ["&hit=1", ""] # avoid redirect
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    size:
+      selector: td:nth-last-child(7)
+    date:
+      selector: td:nth-last-child(3)
+    seeders:
+      selector: td:nth-last-child(6)
+    leechers:
+      selector: td:nth-last-child(5)
+    downloadvolumefactor:
+      case:
+        img[src="images/free.png"]: 0
+        "*": 1
+    uploadvolumefactor:
+      text: 1
+    minimumratio:
+      text: 1.0
+    minimumseedtime:
+      # 2 days (as seconds = 2 x 24 x 60 x 60)
+      text: 172800
```

Comment:
are you able to identify an engine?
e.g. # u-235 v2
otherwise we usually tag # engine n/a if one cannot be determined
also we end the yaml with a blank line.